spec/rails_helper.rb

```diff
@@ -6,8 +6,11 @@
 require 'spec_helper'
 require 'rspec/rails'
 require 'helpers'
+require 'capybara/rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.